<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Falcon-View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="tbd">
      <property name="baseSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:large; font-weight:600;&quot;&gt;Prototype for linuxcnc with dockable widgets.&lt;/span&gt;&lt;/h3&gt;&lt;p&gt;Each dockable can be dragged to different position. See &lt;a href=&quot;https://doc.qt.io/qt-5/qdockwidget.html#details&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;QT documentation&lt;/span&gt;&lt;/a&gt; for further information.&lt;/p&gt;&lt;p&gt;The &lt;span style=&quot; font-weight:600;&quot;&gt;DRO dockable&lt;/span&gt; is special. It contains switchable dro for &lt;span style=&quot; font-style:italic;&quot;&gt;relative&lt;/span&gt; and &lt;span style=&quot; font-style:italic;&quot;&gt;absolute&lt;/span&gt; position. Furthermore the &lt;span style=&quot; font-style:italic;&quot;&gt;fontsize changes&lt;/span&gt; with the hight of the widget.&lt;/p&gt;&lt;p&gt;If it can't be dragged to the desired position, drop it at the desktop. Then shrink its size to minimum and then try again to drop to that location.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionAbsPos"/>
    <addaction name="separator"/>
    <addaction name="actionFrontView"/>
    <addaction name="actionrightView"/>
    <addaction name="actionleftView"/>
    <addaction name="actionBackView"/>
    <addaction name="actionTopView"/>
    <addaction name="action3D_View"/>
    <addaction name="separator"/>
    <addaction name="actionDockables"/>
    <addaction name="actionJog_Simulator"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string comment="what">File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuMode"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>open gcode file</string>
   </property>
   <property name="statusTip">
    <string>open GCode file in editor</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>exit application and shutdown backend</string>
   </property>
  </action>
  <action name="actionAbsPos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Absolute Position</string>
   </property>
   <property name="toolTip">
    <string>toggle between absolute and relative position</string>
   </property>
   <property name="statusTip">
    <string>choose whether to display absolute/relative Coordinates</string>
   </property>
  </action>
  <action name="actionDockables">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dockables</string>
   </property>
  </action>
  <action name="actionFrontView">
   <property name="text">
    <string>FrontView</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionrightView">
   <property name="text">
    <string>rightView</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionleftView">
   <property name="text">
    <string>leftView</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionBackView">
   <property name="text">
    <string>BackView</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionTopView">
   <property name="text">
    <string>TopView</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="action3D_View">
   <property name="text">
    <string>3D-View</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string notr="true">Alt+A</string>
   </property>
  </action>
  <action name="actionJog_Simulator">
   <property name="text">
    <string>Jog-Simulator</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
